src/goExtraInfo.ts

```diff
@@ -28,7 +28,12 @@ export class GoHoverProvider implements HoverProvider {
 			} else {
 				text = lines[0];
 			}
-			let hover = new Hover({ language: 'go', value: text });
+			let hoverTexts = [];
+			if (definitionInfo.doc != null) {
+				hoverTexts.push({ language: null, value: definitionInfo.doc});
```

Comment:
I think that passing this as a string will format in a slightly different way which matches what other extensions do: https://github.com/Microsoft/vscode/search?utf8=%E2%9C%93&q=%22new+Hover%22